sdk/batch/Microsoft.Azure.Batch/src/GeneratedProtocol/Models/PoolSpecification.cs

```diff
@@ -159,15 +160,15 @@ public PoolSpecification(string vmSize, string displayName = default(string), Cl
         public VirtualMachineConfiguration VirtualMachineConfiguration { get; set; }
 
         /// <summary>
-        /// Gets or sets the maximum number of Tasks that can run concurrently
-        /// on a single Compute Node in the Pool.
+        /// Gets or sets the number of task slots that can be used to run
+        /// concurrent tasks on a single compute node in the pool.
         /// </summary>
         /// <remarks>
         /// The default value is 1. The maximum value is the smaller of 4 times
-        /// the number of cores of the vmSize of the Pool or 256.
+        /// the number of cores of the vmSize of the pool or 256.
```

Comment:
We can govern this in the setter.

Reply:
This model is purely autogenerated from swagger. Generally speaking it is best to update the actual Swagger defined RestAPI documentation, because any change here will just get replaced each generation.